### DISCUSSION TITLE
Error with generating morpho

### DISCUSSION THREAD

**Discussion by asmaaharfoush** [neperfepx/neper#294] [Q&A] [answered]
Hello, I used to generate microstructure using this code and suddenly this stop working and got this error
Error  : 10.tess: no such file. I am not sure where I am mistaken, it was perfectly working previously. Can you help me with that? 

```
neper -T -n from_morpho -morpho 'diameq:normal(0.05,0.01)' -o 10 -id 10 -dim 2 -domain 'square(1,1):rotate(0,0,1,45):scale(1.0,1.0,1)' -transform 'scale(1.0,1.0,1),rotate(0,0,1,-45 ),resetlabels'
neper -V 10.tess -cameraangle 10 -imagesize 1024:1024 -dataedgerad 0.001 -print 10
```

Thanks!

**Comment by smallrain1996** [comment, marked as answer]
I ran your code without any problems, otherwise you should reinstall Neper and make sure it is working

**Reply by rquey** [reply]
Same here. Please provide a full bug report if the problem persists.

**Comment by asmaaharfoush** [comment]
Thanks I think this got fixed!
